core/domain/acl_decorators_test.py

```diff
@@ -1144,6 +1144,45 @@ def test_normal_user_can_suggest_changes(self):
         self.logout()
 
 
+class SuggestChangesTest(test_utils.GenericTestBase):
```

Comment:
Maybe, we can improve the naming a bit -- SuggestChangesDecoratorsTest, SuggestionDecoratorsTest or something similar?

Reply:
Done